.github/workflows/ci.yml

```diff
@@ -50,4 +50,4 @@ jobs:
         run: cargo clippy --features dashmap,arrayvec,simd_json,halfbrown,parking_lot,serde_json,mini_moka,hashbrown,secrecy,chrono,nonmax,time,url,extract_map_01 -- --deny warnings
 
       - name: Run Cargo clippy with details, all libs
-        run: cargo clippy --all-features -- --deny warnings
+        run: cargo clippy --features details,dashmap,arrayvec,simd_json,halfbrown,parking_lot,serde_json,mini_moka,hashbrown,secrecy,chrono,nonmax,time,url,extract_map_01 -- --deny warnings
```

Comment:
Why?

Reply:
I thought that adding `saturating_impls` feature (implied by `--all-features` here) would cause a clippy failure due to the MSRV lint, but I may be wrong. I'll revert this.

Reply:
If it doesn't, it's a clippy bug, but when that bug is fixed I'll add an `allow`.